tests/test_Manifest.py

```diff
@@ -0,0 +1,179 @@
+import sys
+import json
+import os
+
+import pytest
+
+from rsconnect.bundle import Manifest
+from rsconnect.exception import RSConnectException
+
+cur_dir = os.path.dirname(__file__)
+html_manifest_json_file = os.path.join(cur_dir, "testdata", "Manifest", "html_manifest.json")
+
+
+@pytest.mark.skipif(sys.platform in ("win32", "win64"), reason="Backslash vs forward slash")
+def test_Manifest_from_json():
+    html_manifest_dict = {
+        "version": 1,
+        "metadata": {"appmode": "static", "primary_html": "index.html", "entrypoint": "index.html"},
+        "files": {
+            "index.html": {"checksum": "c14bd63e50295f94b761ffe9d41e3742"},
+            "test1.txt": {"checksum": "3e7705498e8be60520841409ebc69bc1"},
+            "test_folder1/testfoldertext1.txt": {"checksum": "0a576fd324b6985bac6aa934131d2f5c"},
+        },
+    }
+    manifest_json_str = json.dumps(html_manifest_dict, indent=2)
+    m = Manifest.from_json(manifest_json_str)
+    assert m.json == manifest_json_str
+
+
+@pytest.mark.skipif(sys.platform not in ("win32", "win64"), reason="Backslash vs forward slash")
+def test_Manifest_from_json_Windows():
+    html_manifest_dict = {
+        "version": 1,
+        "metadata": {"appmode": "static", "primary_html": "index.html", "entrypoint": "index.html"},
+        "files": {
+            "index.html": {"checksum": "c14bd63e50295f94b761ffe9d41e3742"},
+            "test1.txt": {"checksum": "3e7705498e8be60520841409ebc69bc1"},
+            "test_folder1\\testfoldertext1.txt": {"checksum": "0a576fd324b6985bac6aa934131d2f5c"},
```

Comment:
Thank you for adding the test. I've been thinking a little more about this. Is it possible for a Windows user to deploy using a manifest created by a Linux user, or vice versa? Should the manifest json always use slash-delimited paths?

Reply:
Separate PR for sure.